### PR TITLE
EXOActiveSyncMailboxPolicy

### PR DESCRIPTION
**Pull request by kasaxena5**
#### Pull Request (PR) description
ActiveSyncMailboxPolicy

### CODE REVIEW

**Comment by NikCharlebois** [comment]
We need to fix the EXO example, but also do a merge with upstream to fix the Intune related items.

**Comment by FabienTschanz** [comment]
@kasaxena5 Can you please update the branch or close the PR? Thank you.

**Comment by kasaxena5** [comment]
> @kasaxena5 Can you please update the branch or close the PR? Thank you.

Updated the branch with Examples and Unit Tests, Please take a look

File: Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/1-Create.ps1 [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            AllowSMIMESoftCerts                      = $True;
+            AllowStorageCard                         = $True;
+            AllowTextMessaging                       = $True;
+            AllowUnsignedApplications                = $True;
+            AllowUnsignedInstallationPackages        = $True;
+            AllowWiFi                                = $True;
+            AlphanumericDevicePasswordRequired       = $False;
+            ApprovedApplicationList                  = @();
+            AttachmentsEnabled                       = $True;
+            Credential                               = $Credscredential;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/1-Create.ps1]: Credentials is not passed as a parameter. We need only appId, tenantId and certThumbprint

File: Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/2-Update.ps1 [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            AllowSMIMESoftCerts                      = $True;
+            AllowStorageCard                         = $True;
+            AllowTextMessaging                       = $True;
+            AllowUnsignedApplications                = $True;
+            AllowUnsignedInstallationPackages        = $True;
+            AllowWiFi                                = $True;
+            AlphanumericDevicePasswordRequired       = $False;
+            ApprovedApplicationList                  = @();
+            AttachmentsEnabled                       = $True;
+            Credential                               = $Credscredential;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/2-Update.ps1]: Credentials is not passed as a parameter. We need only appId, tenantId and certThumbprint

File: Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/3-Remove.ps1 [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            AllowSMIMESoftCerts                      = $True;
+            AllowStorageCard                         = $True;
+            AllowTextMessaging                       = $True;
+            AllowUnsignedApplications                = $True;
+            AllowUnsignedInstallationPackages        = $True;
+            AllowWiFi                                = $True;
+            AlphanumericDevicePasswordRequired       = $False;
+            ApprovedApplicationList                  = @();
+            AttachmentsEnabled                       = $True;
+            Credential                               = $Credscredential;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/3-Remove.ps1]: Credentials is not passed as a parameter. We need only appId, tenantId and certThumbprint

**Review by NikCharlebois — CHANGES_REQUESTED** [review]
Please update the examples

File: Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/1-Create.ps1 (hunk 1 above)
Comment (kasaxena5) [reply Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/1-Create.ps1]: Removed

File: Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/2-Update.ps1 (hunk 2 above)
Comment (kasaxena5) [reply Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/2-Update.ps1]: Removed

File: Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/3-Remove.ps1 (hunk 3 above)
Comment (kasaxena5) [reply Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/3-Remove.ps1]: Removed

**Review by NikCharlebois — APPROVED** [review]

**Pull request merged**